app/main.py

```diff
@@ -1,8 +1,29 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+
+        Person.people[name] = self
+
+    def __repr__(self) -> str:
+        return (
+            f"Person(name={self.name}, "
+            f"age={self.age}, "
+            f"wife={self.wife and self.wife.name}, "
+            f"husband={self.husband and self.husband.name})"
```

Comment:
The attributes `wife` and `husband` are used in the `__repr__` method without being initialized in the `__init__` method. This could lead to an `AttributeError` if these attributes are accessed before being set. Consider initializing these attributes in the `__init__` method with a default value, such as `None`.